drivers/media/i2c/imx477.c

```diff
@@ -1235,21 +1235,9 @@ static u32 imx477_get_format_code(struct imx477 *imx477, u32 code)
 
 static void imx477_set_default_format(struct imx477 *imx477)
 {
-	struct v4l2_mbus_framefmt *fmt = &imx477->fmt;
-
 	/* Set default mode to max resolution */
 	imx477->mode = &supported_modes_12bit[0];
-
-	fmt->code = MEDIA_BUS_FMT_SRGGB12_1X12;
-	fmt->colorspace = V4L2_COLORSPACE_SRGB;
-	fmt->ycbcr_enc = V4L2_MAP_YCBCR_ENC_DEFAULT(fmt->colorspace);
-	fmt->quantization = V4L2_MAP_QUANTIZATION_DEFAULT(true,
-							  fmt->colorspace,
-							  fmt->ycbcr_enc);
-	fmt->xfer_func = V4L2_MAP_XFER_FUNC_DEFAULT(fmt->colorspace);
```

Comment:
Do these other fields get set to sensible values still?

Reply:
fmt->code is the only field that gets referenced anywhere in the code.

The colourspace gets set in `imx477_update_image_pad_fmt` and `imx477_reset_colourspace`

Reply:
OK, I'm happy. The fact that the variable has been renamed from ```imx477->fmt``` to ```imx477->fmt_code``` and it still works does imply that it's not used anywhere else.